crates/path/src/builder.rs

```diff
@@ -89,6 +89,9 @@ use crate::{EndpointId, Winding};
 use std::iter::IntoIterator;
 use std::marker::Sized;
 
+/// https://spencermortensen.com/articles/bezier-circle/
+const BEZIER: f32 = 0.55191505;
```

Comment:
This is quite specific to the logic of the bezier approximation so I would rather keep this near the code that uses it otherwise it is hard to understand what it is about (especially with a name like BEZIER). You could roll it back to how it was (declared twice), but using a `const` instead of a regular immutable variable.